react-counter/src/components/Refresh.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Refresh({reload}) {
+  return (
+    <div className='refresh' onClick={reload}>

```

Comment:
Please, name your callback in more appropriate way (for example `handleMinusClick` or something like that). We usually name callbacks starting with `handle..` if the event starts with `on..` (`onClick`, `onDrag`, etc.)